lib/openhab/dsl/items/switch_item.rb

```diff
@@ -20,6 +20,23 @@ def truthy?
           on?
         end
 
+        #
```

Comment:
I would highly recommend using format_type instead of overriding command directly. otherwise it won't be able to work with `ensure`

Reply:
I think it did work with ensure when I printed the ancestors it seems correct.

I will check out format command as that might cleaner.